src/preprocessing.ml

```diff
@@ -584,42 +641,41 @@ let lint_module sctx ~dir ~expander ~dep_kind ~lint ~lib_name ~scope ~dir_kind =
           if staged then
             Errors.fail loc
               "Staged ppx rewriters cannot be used as linters.";
-          let flags = List.map ~f:(Expander.expand_str expander) flags in
-          let args : _ Arg_spec.t =
-            S [ As flags
-              ; As (cookie_library_name lib_name)
-              ]
-          in
           let corrected_suffix = ".lint-corrected" in
-          let driver_and_flags =
+          let driver_flags_and_args =
```

Comment:
I'd keep this as `driver_and_flags`. `driver_flags_and_args` reads as "driver flags and args" rather than "driver, flags and args". Additionally, a similar variable is still called `driver_and_flags` a bit after in this file